CONTRIBUTING.md

```diff
@@ -2,7 +2,13 @@
 
 ## Generate Protobuf Go Files
 
-Make sure you have Docker installed.
+1. Make sure `opamp-go/internal/opamp-spec` submodule files exist. You can do this
+either by cloning the `opamp-go` repo with submodules the first time, e.g. 
+`git clone --recurse-submodules git@github.com:open-telemetry/opamp-go.git`.
+Alternatively, if you already cloned `opamp-go` without the submodule then
+execute `git submodule update --init` to fetch all files for `opamp-spec` submodule.
 
-Run `make gen-proto`. This should compile `internal/proto/*.proto` files to 
+2. Make sure you have Docker installed.
+
+3. Run `make gen-proto`. This should compile `internal/proto/*.proto` files to 
```

Comment:
We could point out that the current setup is tested for `linux/amd64` and is known to NOT work on M1 chips etc. Just to safe folks from wasting time?

Reply:
Done.